libcxx/utils/libcxx/test/format.py

```diff
@@ -188,6 +189,7 @@ def parseScript(test, preamble):
             "%dbg(MODULE std) %{cxx} %{flags} "
             f"{compileFlags} "
             "-Wno-reserved-module-identifier -Wno-reserved-user-defined-literal "
+            "-fexperimental-modules-reduced-bmi "
```

Comment:
If this is the "new way" of building with C++20 modules, then I think it would make sense to mark the offending libc++ tests as `UNSUPPORTED: clang-18, clang-19`. But that does seem like a big user-facing change too, doesn't it?

Reply:
Yeah, so I'd like to accept @mizvekov 's suggestion to not emit the warning